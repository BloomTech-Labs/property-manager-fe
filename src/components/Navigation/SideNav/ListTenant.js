import React from 'react';
import ListItem from '@material-ui/core/ListItem';
import ListItemIcon from '@material-ui/core/ListItemIcon';
import ListItemText from '@material-ui/core/ListItemText';
// import AccountCircleIcon from '@material-ui/icons/AccountCircle'; -- Giving Tenant a small difference in looks.
import PersonRoundedIcon from '@material-ui/icons/PersonRounded';
import { navigate } from '@reach/router';

function ListView() {
  return (
    <div>
      <ListItem button onClick={() => navigate('/dashboard/profile')}>
        <ListItemIcon>
          <PersonRoundedIcon />
        </ListItemIcon>
        <ListItemText primary="Profile" />
      </ListItem>
    </div>
  );
}

export default ListView;
