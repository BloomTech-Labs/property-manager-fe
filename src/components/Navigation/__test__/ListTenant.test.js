import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ListTenant from '../SideNav/ListTenant';

afterEach(rtl.cleanup);

test('snapshot ListTenant', () => {
  const { wrapper } = rtl.render(<ListTenant />);
  expect(wrapper).toMatchSnapshot();
});

test('Renders', () => {
  const { wrapper } = rtl.render(<ListTenant />);
  expect(wrapper).not.toBeNull();
});
