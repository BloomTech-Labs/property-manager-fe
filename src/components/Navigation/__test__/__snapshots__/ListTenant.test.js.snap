// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot ListTenant 1`] = `undefined`;
